playwright-e2e/data/fake-user.json

```diff
@@ -34,6 +34,31 @@
     "state": "MA",
     "zip": "02114"
   },
+  "adult": {
```

Comment:
`adult` and `patenti` data are idential in `fake-user.json`. I will remove `patient` data in a separate new PR because some existing E2E tests depends it.